app/views/testing/index.html.erb

```diff
@@ -2,32 +2,42 @@
 <div>
   <%= form_tag("/testing/review", method: :post) do %>
     <div id='userinfo'>
-      Broswer Version 
-      <%= text_field_tag("userinfo[broswer_version]", "#{@browser.name} #{@browser.full_version}", class: "form-control label-padding") %>
-      OS Version 
-      <%= text_field_tag("userinfo[os_version]", "#{@browser.platform}", class: "form-control label-padding") %>
-      <%= link_to("GitHub Personal Access Token", "https://github.com/settings/applications") %>:
-      <%= text_field_tag("userinfo[auth_token]", "", class: "form-control label-padding") %>
+      Broswer Version
+      <%= text_field_tag("userinfo[broswer_version]",
+          "#{@browser.name} #{@browser.full_version}",
+          class: "form-control label-padding") %>
+      OS Version
+      <%= text_field_tag("userinfo[os_version]", "#{@browser.platform}",
```

Comment:
these are strings, right?
why are we embedding strings in strings? can this just be `@browser.platform`